op_crates/file/lib.rs

```diff
@@ -1,7 +1,85 @@
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
 
+use deno_core::error::null_opbuf;
+use deno_core::error::AnyError;
+use deno_core::url::Url;
 use deno_core::JsRuntime;
+use deno_core::ModuleSpecifier;
+use deno_core::ZeroCopyBuf;
+use std::collections::HashMap;
 use std::path::PathBuf;
+use std::sync::Arc;
+use std::sync::Mutex;
+use uuid::Uuid;
+
+#[derive(Clone)]
+pub struct Blob {
+  pub data: Vec<u8>,
+  pub r#type: String,
```

Comment:
This is a very confusing property name.  Is there a good reason it is named something so obtuse and with a `#` in the name, unlike any other property on a struct?  While I realise it is named `.type` in the browser, I don't see why we wouldn't name it `media_type` in Rust.

Reply:
The `r#` is rust syntax - it isn't actually part of the property name. It is the raw string token and just tells rust to not interpret the following property name as a keyword, instead just interpret it as a raw string (https://doc.rust-lang.org/reference/tokens.html#characters-and-strings). I will update it to `media_type` anyway to simplify :-)